src/Datastore/RAMDataStore.cpp

```diff
@@ -241,15 +241,15 @@ TResult<Queue> RAMDataStore::getQueue(QueueType q) {
   return (const Queue)(*pQueue);
 }
 
-TResult<PlaybackTrack> RAMDataStore::getPlayingTrack() {
+TResult<QueuedTrack> RAMDataStore::getPlayingTrack() {
```

Comment:
If we do not use `PlaybackTrack` for whatever reason, we should use `BaseTrack` because `QueuedTrack` has nothing `mCurrentTrack` would benefit of.

Reply:
Votes? the client might want to see that

Reply:
for the currently playing track votes doesn't matter because it is already playing.

other than that, there is the interface specification (which all clients agreed upon) which does not specify the vote count for the already playing track

Reply:
i think at this point in the projects' progress we should stick to the interface spec, since client teams are already implementing their side of things.